File: app/src/main/java/com/octopus/android/multimedia/MultimediaApplication.java
package com.octopus.android.multimedia;

import android.content.Intent;
import android.os.Build;

import com.zhuchao.android.session.MApplication;

public class MultimediaApplication extends MApplication {

    @Override
    public void onCreate() {
        super.onCreate();
        ///MMLog.d("MultimediaApplication", "MultimediaApplication onCreate!");
        /*if (MultimediaService.mThis == null) {
            Intent intent1 = new Intent(appContext, MultimediaService.class);
            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
                startForegroundService(intent1);
            } else {
                startService(intent1);
            }
        }*/
    }

    @Override
    public void onLowMemory() {
        super.onLowMemory();
    }

    @Override
    public void onTrimMemory(int level) {
        super.onTrimMemory(level);
    }

    @Override
    public void registerActivityLifecycleCallbacks(ActivityLifecycleCallbacks callback) {
        super.registerActivityLifecycleCallbacks(callback);
    }

    @Override
    public void onTerminate() {
        super.onTerminate();
    }

}

File: app/src/main/java/com/octopus/android/multimedia/MultimediaApplication.kt
package com.octopus.android.multimedia;

import android.content.Context
import android.content.Intent
import android.content.IntentFilter
import android.hardware.usb.UsbManager
import android.os.Build
import com.airbnb.mvrx.Mavericks
import com.zhuchao.android.fbase.MMLog
import com.zhuchao.android.session.Cabinet
import com.zhuchao.android.session.MApplication

class MultimediaApplication : MApplication() {

    override fun onCreate() {
        super.onCreate()
        MMLog.d("MultimediaApplication", "MultimediaApplication onCreate!")
        Mavericks.initialize(this)
        //////////////////////////////////////////////////////////////////////////////////
    }

    override fun onTerminate() {
        super.onTerminate()
    }
}
